pages/docs/reference/opt-in-requirements.md

```diff
@@ -7,9 +7,14 @@ title: "Opt-in Requirements"
 
 # Opt-in Requirements
 
-> The opt-in requirement annotations `@RequiresOptIn` and `@OptIn` are *experimental* in Kotlin 1.3.
+> The opt-in requirement annotations `@RequiresOptIn` and `@OptIn` are *experimental*. 
 > See the usage details [below](#experimental-status-of-the-opt-in-requirements).
-{:.note}
+{:.note} 
+
+> Since 1.3.70, `@RequiresOptIn` and `@OptIn` replace  `@Experimental` and `@UseExperimental` respectively;
```

Comment:
Hmm, still not quite clear that they were _introduced_ only in 1.3.70.
Perhaps, "RequireOptIn and OptIn annotations were introduced in 1.3.70 to replace previously used Experimental and UseExperimental"?